client/src/utils/i18nHelper.ts

```diff
@@ -5,13 +5,28 @@ export function format(src: string, ...args: string[]): string {
 }
 
 // Load the i18n function
-let trHelper: (a: string) => string;
+let trHelper: (a: string) => string | string[];
 _.subscribe((a) => (trHelper = a));
 
 // Prefer to use 'text-transform: capitalize' css attribute for capitalization
 export function tr(a: string, capitalize = false): string {
-    const cap = capitalize
-        ? (s: string) => s.charAt(0).toUpperCase() + s.slice(1)
-        : (s: string) => s;
-    return cap(trHelper(a));
+    try {
+        const cap = capitalize
+            ? (s: string) => s.charAt(0).toUpperCase() + s.slice(1)
+            : (s: string) => s;
+        return cap(trHelper(a) as string);
+    } catch (err) {
+        return 'A venir';
+    }
+}
```

Comment:
This return "A venir" in the UI.
+ no error message

Reply:
This is the wanted behaviour.

Reply:
Yes, but the text is present, why "à venir" ?